### ISSUE TITLE
getting a list of everyone the user follows

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#32]
# Proposed change
- First need to get the static information about all the streams the user follows

# Why is this important
- This will ultimately be the version that individuals see

# Additional context
- I want it to look something:

<img width="181" alt="TwitchFollowingExample" src="https://github.com/thePlebDev/Clicker/assets/47083513/7af688b0-188b-4a12-be94-a364c240a57e">

**Comment by thePlebDev** [comment]
- Getting all the user's followed live streamers are found [HERE](https://dev.twitch.tv/docs/api/reference/#get-streams)

**Comment by thePlebDev** [comment]
- a basic example request is this:
```
curl -X GET 'https://api.twitch.tv/helix/streams/followed?user_id=141981764' \
-H 'Authorization: Bearer 2gbdx6oar67tqtcmt49t3wpcgycthx' \
-H 'Client-Id: wbmytr93xzw8zbg0p1izqyzzc5mbiz'

```
- So it looks like we need the `client_id` that is retured when we validate the user

**Comment by thePlebDev** [comment]
- closed with #35
